src/transforms/aggregate.rs

```diff
@@ -26,6 +26,7 @@ pub struct AggregateConfig {
     ///
     /// During this time frame, metrics with the same series data (name, namespace, tags, and so on) are aggregated.
     #[serde(default = "default_interval_ms")]
+    #[configurable(metadata(docs::human_name = "Interval Time Frame"))]
```

Comment:
"Interval Time Frame" feels a bit repetitive, but it also doesn't fit the pattern of "X Interval" that similar fields have been changed to.

How about "Flush Interval"?

Reply:
"Flush Interval" is much better!